neonvm/hack/kernel/linux-config-6.1.92

```diff
@@ -2554,7 +2554,7 @@ CONFIG_QUOTA=y
 CONFIG_PRINT_QUOTA_WARNING=y
 # CONFIG_QUOTA_DEBUG is not set
 # CONFIG_QFMT_V1 is not set
-# CONFIG_QFMT_V2 is not set
+CONFIG_QFMT_V2=y
```

Comment:
Could you add the rationale for this to the PR description?

Reply:
Quotas don't work without this.